plugins/services/src/js/constants/ServiceActionLabels.js

```diff
@@ -0,0 +1,10 @@
+const ServiceActionLabels = {
+  delete: "Delete",
+  restart: "Restart",
+  resume: "Resume",
+  scale: "Scale",
+  scale_group: "Scale By",
```

Comment:
I wonder if `scale_by` wouldn't make a bit more consistent, what you think? not a big deal though.